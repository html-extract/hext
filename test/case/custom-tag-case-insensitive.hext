<yt-formatted-string class:c @text:x />
