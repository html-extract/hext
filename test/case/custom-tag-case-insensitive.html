<div>
  <yt-formatted-string class="one">Content1</yt-formatted-string>
  <yt-fOrmatted-string class="two">Content2</yt-fOrmatted-string>
  <Yt-Formatted-String class="three"></Yt-Formatted-String>
  <NOMATCH class="no">no match</NOMATCH>
</div>
