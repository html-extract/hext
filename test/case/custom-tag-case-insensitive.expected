{"c":"one","x":"Content1"}
{"c":"three","x":""}
{"c":"two","x":"Content2"}
